okteto.yml

```diff
@@ -0,0 +1,13 @@
+name: hugo
+image: fluxcd/website:hugo-support
```

Comment:
How are users suppose to use this? There is no such image on Docker Hub.

Reply:
We will have to publish it there. It does not need to change often, but if we think it needs more scaffolding so this image can be kept up to date, ...